skimage/restoration/_denoise_cy.pyx

```diff
@@ -111,6 +110,7 @@ def _denoise_tv_bregman(np_floats[:, :, ::1] image, np_floats weight,
     shape_ext = (rows2, cols2, dims)
 
     cdef:
+        np_floats[:, :, ::1] cu = u
```

Comment:
This line is causing the CI failures: `u` is not declared and `cu` is not used afterward...
Simply removing this line may solve the problem.